IoTCoreDefaultApp/IoTCoreDefaultApp/Views/Settings.xaml

```diff
@@ -178,7 +223,8 @@
                     <RowDefinition Height="80"/>
                     <RowDefinition Height="Auto"/>
                     <RowDefinition Height="Auto"/>
-                    <RowDefinition Height="*"/>
+                    <RowDefinition Height="117*"/>
+                    <RowDefinition Height="172*"/>
```

Comment:
just want to make sure that people using small screens can still use the bt functionality